### DISCUSSION TITLE
Could you add the new stickers from the newest sticker capsule?

### DISCUSSION THREAD

**Discussion by Sarahsqueaks** [danielkrupinski/Osiris#3241] [Ideas]
I love the new stickers, would look really good on a printstream, would be cool if you could add them.

**Comment by KurastikO** [comment]
Every new sticker or skin is already implemented in the inventory changer because it uses a method where it scans the .vpk files and therefore no need to manually patch skins and stuff

**Reply by aoqia194** [reply]
This.
